### PR TITLE
docs: add sample manifests for Backend Service

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/manifest/backend-service.en.md [hunk 1]
```diff
@@ -1,63 +1,142 @@
...
+        ```yaml
+            # Your service is reachable at "http://api.${COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080" only within your VPC.
+            name: api
+            type: Backend Service
+
+            image:
+              build: ./api/Dockerfile
+              port: 8080
+              healthcheck:
+                command: ["CMD-SHELL", "curl -f http://localhost:8080 || exit 1"]
```
Comment (dannyrandall) [inline site/content/docs/manifest/backend-service.en.md:16]: do we actually need `|| exit 1`? doesn't curl already fail on any non-2xx with `-f`?

File: site/content/docs/manifest/backend-service.en.md [hunk 2]
```diff
@@ -1,63 +1,142 @@
...
+        type: Backend Service
+    
+        image:
+          build: ./api/Dockerfile
+          port: 8080
+
+        http:
+          path: '/'
+          alias: 'v1.api.example.com'
+          hosted_zone: AN0THE9H05TED20NEID # Insert record for v1.api.example.com to the hosted zone.
```
Comment (dannyrandall) [inline site/content/docs/manifest/backend-service.en.md:91]: hahaha clever

Comment (efekarakus) [reply site/content/docs/manifest/backend-service.en.md:91]: That wasn't me! This feels like a @huanjani thing to do

File: site/content/docs/manifest/backend-service.en.md (hunk 1 above)
Comment (efekarakus) [reply site/content/docs/manifest/backend-service.en.md:16]: 🤷 https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html it was just copy/pasted from the ECS docs

**Review by dannyrandall — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]
:shipit:

**Pull request merged**